src/ucp/api/ucp.h

```diff
@@ -1600,6 +1642,15 @@ typedef struct ucp_mem_map_params {
       *    internally.
       */
      ucs_memory_type_t      memory_type;
+
+     /**
+      * Exported memory handle buffer with which a local memory handle should
```

Comment:
```
Exported memory handle buffer as retuned by @ref ... for a memory handle created with @ref .. flag
When this field is specified for @ref ucp_mem_map, the resulting memory handle would be mapping peer memory instead of local memory.
```

Reply:
done